sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/CommandHandler.scala

```diff
@@ -27,7 +27,8 @@ case class CommandHandler(
     grpcMethodName: String,
     messageCodec: SpringSdkMessageCodec,
     requestMessageDescriptor: Descriptors.Descriptor,
-    methodInvokers: Map[String, MethodInvoker]) {
+    methodInvokers: Map[String, MethodInvoker],
+    ignoreUnknown: Boolean) {
```

Comment:
I feel that this option should not be here, but only at router level. We can move it to the component descriptor and then pass to the router.

Can have two variants of `lookupInvoker`? 

`lookInvoker` can stay as in this PR, returning `Option`, but not deciding if it throws exception based on `ignoreUnknown`. The router should make that decision. 

Then we also introduce a strict lookup. Can be `getInvoker`. This method would not return an `Option`, but a non-wrapped invoker. We will use this method in the routers that always expect to find an invoker, like the entity routers.

Reply:
refactored

Reply:
done now